packages/dds/task-manager/src/interfaces.ts

```diff
@@ -5,19 +5,126 @@
 
 import { ISharedObject, ISharedObjectEvents } from "@fluidframework/shared-object-base";
 
+/**
+ * Describes the event listener format for {@link ITaskManagerEvents} events.
+ *
+ * @param taskId - The unique identifier of the related task.
+ */
+export type TaskEventListener = (taskId: string) => void;
+
 /**
  * Events emitted by {@link TaskManager}.
  */
 export interface ITaskManagerEvents extends ISharedObjectEvents {
 	/**
-	 * Notifies when the local client has reached the front of the queue, left the queue, or a task was completed.
-	 * Does not account for known pending ops, but instead only reflects the current state.
+	 * Notifies when the specified task has been exclusively assigned to the client.
+	 *
+	 * @remarks Does not account for known pending ops, but instead only reflects the current state.
+	 *
+	 * @eventProperty
+	 */
+	(event: "assigned", listener: TaskEventListener);
+
+	/**
+	 * Notifies when the specified task has been completed.
+	 *
+	 * @eventProperty
+	 */
+	(event: "completed", listener: TaskEventListener);
+
+	/**
+	 * Notifies when the specified task has been abandoned (i.e. it will never be completed).
+	 *
+	 * @eventProperty
 	 */
-	(event: "assigned" | "completed" | "lost", listener: (taskId: string) => void);
+	(event: "lost", listener: TaskEventListener);
 }
 
 /**
- * Task manager interface
+ * A distributed data structure that tracks queues of clients that want to exclusively run a task.
```

Comment:
Note: these were moved from the class, but are unchanged aside from migrating away from embedded Markdown syntax, which isn't supported by TSDoc (it happens to work out in select cases on our website, but is not guaranteed). Instead, I've opted to use the `@example` tag syntax.